client/index.html

```diff
@@ -4,28 +4,151 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="shortcut icon" href="./images/favicon.png" type="image/x-icon" />
 		<link rel="preconnect" href="https://fonts.googleapis.com" />
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 		<link
 			href="https://fonts.googleapis.com/css2?family=Fuzzy+Bubbles:wght@400;700&display=swap"
 			rel="stylesheet"
 		/>
+		<script
+			src="https://kit.fontawesome.com/b7207ee847.js"
+			crossorigin="anonymous"
+		></script>
 		<link rel="stylesheet" href="./style.css" />
 		<script defer src="./index.js"></script>
-		<title>Your Stories</title>
+		<title>Mouse Tails</title>
 	</head>
 	<body>
-		<div class="login-container">
-			<form id="login-form">
+		<!-- Login Form -->
+		<div class="form-container" id="login-container">
+			<h1>Login</h1>
+			<form class="form" id="login-form">
 				<label for="username">Username</label>
-				<input type="text" id="username" />
+				<div class="input-group">
+					<input
+						type="text"
+						id="username"
+						placeholder="Enter Username"
```

Comment:
I like how there's placeholders for email, username and password. Very accessible.